maestro/12-side-menu/C202842-side-menu-no-crash.yaml

```diff
@@ -66,11 +66,10 @@ tags:
     id: "sideMenuButton"
 - tapOn:
     id: "sideMenuButton"
-- tapOn: "Share Edge"
-- assertVisible: "Copy"
-- tapOn: "Close"
+#- tapOn: "Share Edge"
+#- assertVisible: "Copy"
+#- tapOn: "Close"
 - tapOn: "Settings"
-- assertVisible: "Settings"
```

Comment:
This assert shouldn't be removed but be changed to something that shows on the initial `Settings` screen